### PR TITLE
Create a constants file to contain the base API URL

### LINKED ISSUE #101: Create a constants file to contain the base API URL
**Is your feature request related to a problem? Please describe.**
- Currently, If we need to change the hostname of the API URLs we need to change every file that contains the API URLs and there will be 19+ file changes. 
- This is not a good approach and we have to face issues when committing the changes

**Describe the solution you'd like**
- Create a file called `constants.ts` in the root 
- Add the base URL as a variable ex: `const API_URL = 'http://localhost:8080/api'`
- Replace the existing API URLs with the variable
ex: `http://localhost:8080/api/programs` will be `${API_URL}/programs`

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose
The purpose of this PR is to fix #101

## Goals
To create a constants file to contain the base API URL

## Approach
Created a constants file with the API URL and replaced the existing API URLs with the variable

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ function App() {
 
   const getUser = () => {
     axios
-      .get('http://localhost:8080/api/me', { withCredentials: true })
+      .get`${API_URL}/me`, { withCredentials: true })
```
Comment (anjula-sack) [inline src/index.tsx]: Missing `(`

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
There are about 20 files containing the API URL and also  run `npm run lint` after `npm i` because the build is failing

**Review by Gimhan-minion — DISMISSED** [review]
Good Job @gajindubandara 💯

File: src/index.tsx (hunk 1 above)
Comment (gajindubandara) [reply src/index.tsx]: fixed!

**Review by anjula-sack — COMMENTED** [review]
Please check again there should be 20 files 
image

**Review by anjula-sack — APPROVED** [review]
Thank you! @gajindubandara

**Pull request merged**